src/utils/server/districtRankings/upsertRankings.ts

```diff
@@ -222,3 +222,34 @@ export async function getDistrictRank(
     score,
   }
 }
+
+export async function getMultipleDistrictRankings(
+  redisKey: (typeof REDIS_KEYS)[keyof typeof REDIS_KEYS] = CURRENT_DISTRICT_RANKING,
+  members: MemberKey[],
```

Comment:
Refactor into object so that there is no need to pass `undefined` for the redisKey

```suggestion
export async function getMultipleDistrictRankings(
  {
    members,
    redisKey = CURRENT_DISTRICT_RANKING,
  }: {
    members: MemberKey[];
    redisKey?: (typeof REDIS_KEYS)[keyof typeof REDIS_KEYS];
  }
```